lambda/src/main/resources/db/migration/V21__change_consignment_metata_foreign_key.sql

```diff
@@ -0,0 +1,13 @@
+ALTER TABLE "ConsignmentMetadata" DROP CONSTRAINT "ConMetadata_Property_fkey";
+ALTER TABLE "ConsignmentMetadata" ALTER COLUMN "PropertyId" TYPE text;
+ALTER TABLE "ConsignmentMetadata" RENAME COLUMN "PropertyId" TO "PropertyName";
```

Comment:
Is this table currently empty? Is that why it's safe to rename a column?

Reply:
Yes. At the moment the Transfer Agreement data is being saved to the `TransferAgreement` table.